custom/shared/contexts/ScreenShareProvider.js

```diff
@@ -0,0 +1,40 @@
+import React, { createContext, useContext, useMemo } from 'react';
+import { useScreenShare as useDailyScreenShare } from '@daily-co/daily-react-hooks';
+import PropTypes from 'prop-types';
+
+export const MAX_SCREEN_SHARES = 2;
+
+const ScreenShareContext = createContext(null);
+
+export const ScreenShareProvider = ({ children }) => {
+  const {
+    isSharingScreen,
+    screens,
+    startScreenShare,
+    stopScreenShare
+  } = useDailyScreenShare();
+
+  const isDisabled = useMemo(() => screens.length >= MAX_SCREEN_SHARES && !isSharingScreen,
```

Comment:
Isn't `screens` an [object](https://docs.daily.co/reference/daily-react-hooks/use-screen-share#screens-properties)?  Will `.length`? work?

Reply:
I think it's an array: https://github.com/daily-co/daily-react-hooks/blob/main/src/hooks/useScreenShare.ts#L59

Reply:
yep, it's an array of screen objects.

Reply:
But then why does [this](https://docs.daily.co/reference/daily-react-hooks/use-screen-share#screens-properties) say object 🥲 is there a different `screens` I should be looking at? Or am I just misunderstanding completely lol

Reply:
Yeah, I think it's a typo in the docs, we should we change that.

Reply:
thanks for clarifying!

Reply:
We should also update the JSON example then. I'll make a Linear ticket.